### ISSUE TITLE
shards install does updates?

### ISSUE THREAD

**Issue by rdp** [crystal-lang/shards#286]
after running a "shards install" I get some updates...expected?  I'd have expected it to respect the shard.lock file.

```

$ git diff
diff --git a/html5_javascript/kemal_server/shard.lock b/html5_javascript/kemal_server/shard.lock
index 7f150a1a..71dd2fb1 100644
--- a/html5_javascript/kemal_server/shard.lock
+++ b/html5_javascript/kemal_server/shard.lock
@@ -2,7 +2,11 @@ version: 1.0
 shards:
   db:
     github: crystal-lang/crystal-db
-    version: 0.5.0
+    version: 0.5.1
+
+  exception_page:
+    github: crystal-loot/exception_page
+    version: 0.1.2
 
   kemal:
     github: sdogruyol/kemal
@@ -22,5 +26,5 @@ shards:
 
   radix:
     github: luislavena/radix
-    version: 0.3.8
+    version: 0.3.9
```

how to repro:

```
clone https://github.com/rdp/sensible-cinema
cd sensible-cinema/html5_javascript/kemal_server/
git checkout 4d80d0f1b60a56687446f00d959a3917a4a5c123
shards install
```

shards 0.8.1

**Comment by ysbaddaden** [comment]
There is a new shard (exception_page) that got installed.

Probably fixed in 0.9.0 where such cases will choose a solution that has the minimum impact.

**Comment by rdp** [comment]
Seems to work with 0.9.0 thanks!
